paymentapi/src/main/java/fi/hel/verkkokauppa/payment/model/paytrail/payment/FormFieldModel.java

```diff
@@ -0,0 +1,16 @@
+package fi.hel.verkkokauppa.payment.model.paytrail.payment;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+
+@Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class FormFieldModel {
+    @Field(type = FieldType.Text)
+    String name;
```

Comment:
Add private scope to these fields. I know its not used in all the older entity/document classes, but we should use private scope for new classes.

Reply:
Yeh good point, I will do this